source/tutorials/cross-compilation.rst

```diff
@@ -0,0 +1,262 @@
+.. _ref-cross-compilation:
+
+Cross-compilation
+=================
+
+Cross-compilation is the act of **compiling code** on the **build platform**
+to the **host platform**, where the compiled **executable runs**. [#]_
```

Comment:
```suggestion
When compiling code, we can distinguish between the **build platform**, where the executable
is *built*, and the **host platform**, where the compiled executable *runs*. [#]_
**Native compilation** is the special case where those two platforms are the same.
**Cross compilation** is the general case where those two platforms are not.
```

Also I wouldn't hyphenate "cross-compilation". It's just a noun phrase. Only "cross-compiled" the adjective as in "let's use this cross-compiled library" needs it.

Reply:
I took the inspiration from nixpkgs manual: https://nixos.org/manual/nixpkgs/stable/#chap-cross